### ISSUE TITLE
use not used key

### ISSUE THREAD

**Issue by hektr** [ranger/ranger#2354]
May be, to add an `>` key to open ranger command line with `:travel `
`>` is not still used in ranger, as I know. And it's very intuitive (shows to right, associated with going/traveling right)

I've done this by adding a line `map > console travel%space` in my `rc.conf` file

**Comment by toonn** [comment]
I'm open to both of these. It'd make the features a little bit more discoverable because people tend to look at cheatsheets and not documentation.

**Comment by hektr** [comment]
Yes, you are right. By the way, official cheat sheet misses info about dot key `.`
It performs command hint for filter_stack command

**Comment by hektr** [comment]
Yes, people don't read docs :) I think, it's because docs are too long. It's needed to write by hands some short help file (about 120-150 strings) with most used commands and key-bindings. I can do start version of such help file, if you want. But it should be reviewed by people who knows ranger and english language better than me :)

**Comment by toonn** [comment]
I'm entirely willing to proofread and correct doc improvements. Take a look at the User Guide on the wiki because that should serve this purpose. Maybe it can use a short introductory section that goes over all the basics?

**Comment by chu4ng** [comment]
>I think, it's because docs are too long. It's needed to write by hands some short help file (about 100-150 lines) with most used commands and key-bindings

_You was reading docs, right?_ Ranger already have `?` for that helps info, `?k` for keybings, `?c` for command, etc.

Also I think this keybind very useful. Already added it to my rc.conf.
Earlier I used the `f` key with `find` command, where scout with `-aetsf` flags, but it useful only for openning file in current dir.

**Comment by hektr** [comment]
I've updated a little official guide :)

**Comment by hektr** [comment]
wow, it is much better now 👍

**Comment by hektr** [comment]
I edited guide in my page here https://github.com/hektr/ranger/wiki/Ranger-guide 
If it is better, feel free to copy-paste it :)

**Comment by hektr** [comment]
I know one more good key-binding.

`map <  eval fm.open_console('shell -rw mount /dev/sdb  ' + fm.thisfile.relative_path.replace("%", "%%"), position=24)`
May be, it can be written other way, idk :)

Key `<` can be used for mounting filesystems. It is also intuitive :)
